Distance.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Objects;
+
+public class Distance {
+
+            /* Napisz klase ktora trzyma informacje o wartosci odleglosci i
+            o rodzaju miary METRY KILOMETRY CENTYMETRY w klasie ma byc metoda ktora
+             porownuje dwa miary medota toString ktora ladnie wypsuize miare np. 10 km i
+             metoda ktora przyjmuje typ miary i zmieni typ miary obiektu na inny
+             */
+
+
+            enum DistanceUnitType
+            {
+                CENTIMETERS,
+                METERS,
+                KILOMETERS
+            }
+
+            private double distance;
+            private DistanceUnitType measure;
+            private double cmMeasure;

```

Comment:
cmMeasure field is redundant i think and we should avoid defining fields that we don't really need.
You could try to implement method 
int getDistanceInUnit(Distance distance, DistanceUnitType unitType);
that will return distance in different unit, but will not modify the object.
You could use that method for comparing distances and changing unit type.